samples/WebApp/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "webapp",
+  "version": "1.0.0",
+  "description": "Gluon sample web application",
+  "main": "Scripts/app.js",
+  "scripts": {
+    "clean-server": "mono ../../packages/FAKE/tools/FAKE.exe build.fsx Clean",
+    "clean-webapp": "node_modules/.bin/rimraf ./bin && mkdir bin",
+    "clean": "npm run clean-server && npm run clean-webapp",
+
+    "build-server": "mono ../../packages/FAKE/tools/FAKE.exe build.fsx Build",
+    "generate-client": "mono '../../src/Gluon.CLI/bin/Release/Gluon.CLI.exe' --out 'Scripts/Generated.ts' --reflect 'bin/SampleApp.exe'",
```

Comment:
Still have some `mono` references here. Need to think how to make these optional when running on Windows.